src/templates/common/media/index.test.tsx

```diff
@@ -1,41 +1,68 @@
 import { render, screen } from '@testing-library/react'
 import '@testing-library/jest-dom/extend-expect'
 import '@testing-library/jest-dom'
-import { MediaImageComponent, MediaImageProps } from '.'
-import { mediaImageDataService } from './dataService'
-import mediaImage from './mockMedia.json'
+import { MediaImageType } from '@/types/index'
+
+import Image from '../image'
+
 // Language: typescript
 // Path: src/components/media/index.test.tsx
-const image = mediaImageDataService(mediaImage, 'full_content_width')
 
-let data: MediaImageProps = {
-  id: image.id,
-  url: image.url,
-  width: image.width,
-  height: image.height,
-  alt: image.alt,
-  title: image.title,
-  styles: image.styles,
-  imageStyle: null,
+const mediaImage: MediaImageType = {
+  id: '3d6716b3-fb66-4e63-9b21-bb9c024129d3',
+  link: {
+    href: 'http://content-build-medc0xjkxm4jmpzxl3tfbcs7qcddsivh.ci.cms.va.gov/sites/default/files/styles/2_1_large/public/2020-08/Raab.jpg?h=d3381009',
+    meta: {
+      linkParams: {
+        width: 100,
+        height: 100,
+      },
+    },
+  },
+
+  alt: 'Smiling man in glasses.',
+  title: '',
+  width: 1299,
+  height: 1512,
+  url: '/sites/default/files/2020-08/Raab.jpg',
+}
+
+function MediaImageComponent({ imageStyle }: MediaImageType) {
```

Comment:
why is this new component in the test vs using the component itself?

Reply:
+1

Reply:
fixed, I had that in temporarily and then forgot all about it.